### PR TITLE
1516 task105 persistent storage

### PR DESCRIPTION
**Pull request by Sanaiqbal-dev**
Task 105 related to Persistent storage is completed.
User Story : https://dev.azure.com/CodeNinjaConsulting/Talent%20Pool/_workitems/edit/1435

### CODE REVIEW

File: portfolio-project-react/src/App.js [hunk 1]
```diff
@@ -1,21 +1,18 @@
-import "./index.css";
+import { useState } from "react";
 import Header from "./components/Header/Header";
 import Content from "./components/Content/Content";
 import Footer from "./components/Footer/Footer";
-
-import { useState } from "react";
+import { IsEditModeEnabled } from "./EditModeContext";
```
Comment (rohaan-hussain) [inline portfolio-project-react/src/App.js:5]: why we have context work in this PR?

Comment (Sanaiqbal-dev) [reply portfolio-project-react/src/App.js:5]: Using context api for storing a state globally was acceptance criteria of a previous task.

File: portfolio-project-react/src/components/About/About.jsx [hunk 2]
```diff
@@ -12,17 +12,53 @@ import {
...
+  }, [email]);
 
   return (
     <div>
       <div className={styles.aboutHeader}>
         <h1 className={styles.titleAbout}>{ABOUT_HEADING}</h1>
         <label>
-          Total Experience: {totalWorkExperience.years} years {totalWorkExperience.months > 0 ? totalWorkExperience.months : 0} months
+          Total Experience: {totalWorkExperience.years} years{" "}
+          {totalWorkExperience.months > 0 ? totalWorkExperience.months : 0}{" "}
```
Comment (rohaan-hussain) [inline portfolio-project-react/src/components/About/About.jsx]: you don't need to show 0 months when user has 0 months experience and the same goes for years if no of years is 0 then only show months

**Review by rohaan-hussain — APPROVED** [review]
Add some suggestions ... please consider that before merging

File: portfolio-project-react/src/App.js (hunk 1 above)
Comment (rohaan-hussain) [reply portfolio-project-react/src/App.js:5]: Its not a part of this Task then why I am seeing this in this PR? it should be merged into the main branch before creating this PR

**Pull request merged**